moveit_ros/planning_interface/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(moveit_ros_planning_interface)
 find_package(moveit_common REQUIRED)
 moveit_package()
 
+find_package(backward_ros)
```

Comment:
I added this to moveit_common.  I think by doing that it is already here from the moveit_package() call on line 6.

Reply:
I'm sorry, I added this by mistake. I'll remove it but I first wait a bit if there is no other changes to carry out.